packages/react-query/src/utils/copy.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs'
+import { dirname, join, resolve } from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+
+const dir = resolve(__dirname, '..', 'dist')
+
+export function copy(version: number): void {
+  const srcDir = join(dir, `v${version}`)
```

Comment:
Changing to `v4.ts` and `v5.ts` will result in no `v${version}` path, causing a `postinstall` error.

```shell
packages/react-query postinstall$ node -e "import('./dist/postinstall.js').catch(e => console.error(e))"
│ TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of URL. Received undefined
│     at fileURLToPath (node:internal/url:1461:11)
│     at file:///workspaces/suspensive/packages/react-query/dist/chunk-L7RTHCDX.js:8:18
│     at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
│     at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
│     at async importModuleDynamicallyWrapper (node:internal/vm/module:432:15) {
│   code: 'ERR_INVALID_ARG_TYPE'
│ }
└─ Done in 64ms
```